src/v2/Apps/Order/__tests__/OrderApp.jest.tsx

```diff
@@ -101,6 +101,12 @@ describe("OrderApp routing redirects", () => {
                   id: "artwork-id",
                   is_acquireable: true,
                   is_offerable: true,
+                  isInquireable: false,
+                  listPrice: {
+                    __typename: "Money",
+                    major: 12000,
+                  },
+                  priceCurrency: "USD",
```

Comment:
suggestion:
good to check if `<ZendeskWrapper />` renders depending on price etc.